cdap-docs/application-templates/source/etl/templates/sinks/real-time/cube.rst

```diff
@@ -16,3 +18,34 @@ sink.
 
 To configure transformation from a StructuredRecord to a CubeFact, the mapping
 configuration is required, following StructuredRecordToCubeFact documentation.
+
+.. rubric:: Use Case
+
+TODO: Fill me out
+
+.. rubric:: Properties
+
+**name:** Name of the Cube dataset. If the Cube does not already exist, one will be created.
+
+**dataset.cube.resolutions:** Aggregation resolutions to be used if new Cube dataset needs to be created.
+See `Cube dataset configuration details <http://docs.cask.co/cdap/current/en/developers-manual/building-blocks/datasets/cube.html#cube-configuration>`__ for more information.
+
+**dataset.cube.properties:** Provide any dataset properties to be used if new Cube dataset
```

Comment:
Provides any dataset properties to be used if a new Cube dataset
needs to be created; provided as a JSON Map. For example, if aggregations are desired on fields 'abc' and 'xyz', the
property should have these values: {"dataset.cube.aggregation.agg1.dimensions":"abc", "dataset.cube.aggregation.agg2.dimensions":"xyz"}.